src/components/homepage/signup/SignUp.js

```diff
@@ -0,0 +1,113 @@
+import React, {Component} from 'react';
+
+import './SignUp.css';
+
+class SignUp extends Component {
+
+    state = {
+        title: '',
+        Username: '',
+        firstname: '',
+        lastname: '',
+        DOB: '',
+        gender: '',
+        password: ''
+    }
+    onSubmit = (e) => {
+        console.log(this.state.username);
+        console.log(this.state.firstname);
+        console.log(this.state.lastname);
+        console.log(this.state.email);
+        console.log(this.state.gender);
+        console.log(this.state.DOB);
+        console.log(this.state.password);
+        e.preventDefault();
+    }
+    handleUsernameChange = (e) => {
+        this.setState({username: e.target.value})
+    }
+    handleFirstnameChange = (e) => {
+        this.setState({firstname: e.target.value})
+    }
+    handleLastnameChange = (e) => {
+        this.setState({lastname: e.target.value})
+    }
+    handleEmailChange = (e) => {
+        this.setState({email: e.target.value})
+    }
+    handleGenderChange = (e) => {
+        this.setState({gender: e.target.value})
+    }
+    handleDOBChange = (e) => {
+        this.setState({DOB: e.target.value})
+    }
+    handlePasswordChange = (e) => {
+        this.setState({password: e.target.value})
+    }
+
+    render() {
+        const {username, firstname, lastname, email, gender, DOB, password} = this.state;
+        return (
+            <div className={'Details'}>
+                <div className='Details-container'>
+                    <header>
+                        <div className={'header-headings'}>
+                            <span> SignUp</span></div>
+                    </header>
+                    <form className='DetailsForm' onSubmit={this.onSubmit}>
+                        <div
+                            className='Details-field'>
+                            <tr>
+                                <td>
+                                    <input id='username' name='username' type='username' value={username}
+                                           placeholder='Username'
+                                           onChange={this.handleUsernameChange}/></td>
+                                <td><input id='firstname' name='firstname' type='firstname' value={firstname}
+                                           placeholder='Firstname'
+                                           onChange={this.handleFirstnameChange}
+                                /></td>
+                            </tr>
+                            <tr>
+                                <td><input id='lastname' name='lastname' type='lastname' value={lastname}
```

Comment:
Please put a space between last and name in the place holder
do the same for firstname